src/databricks/sqlalchemy/dialect/compiler.py

```diff
@@ -0,0 +1,38 @@
+from sqlalchemy.sql import compiler
+
+
+class DatabricksTypeCompiler(compiler.GenericTypeCompiler):
+    """Originally forked from pyhive"""
+
+    def visit_INTEGER(self, type_):
+        return "INT"
+
+    def visit_NUMERIC(self, type_):
+        return "DECIMAL"
+
+    def visit_CHAR(self, type_):
+        return "STRING"
+
+    def visit_VARCHAR(self, type_):
+        return "STRING"
+
+    def visit_NCHAR(self, type_):
+        return "STRING"
+
+    def visit_TEXT(self, type_):
+        return "STRING"
+
+    def visit_CLOB(self, type_):
+        return "STRING"
+
+    def visit_BLOB(self, type_):
+        return "BINARY"
+
+    def visit_TIME(self, type_):
+        return "TIMESTAMP"
+
+    def visit_DATE(self, type_):
```

Comment:
is this meant to be DATE -> TIMESTAMP?

Reply:
I will confirm. Not sure if Databricks has a "dateless" timestamp type.

Reply:
We do in fact have a dateless type (docs [here](https://docs.databricks.com/sql/language-manual/data-types/date-type.html)). I fixed this and added a test for it in [4db1670](https://github.com/databricks/databricks-sql-python/pull/57/commits/4db1670cd0a3011363ace112eca01ccc277ddf65)